From 61869293dbf71862ec1085fdd80938669ea9e72d Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Fri, 8 Jul 2022 14:46:39 +0300
Subject: [PATCH 013/233] [FIO extras] imx: mx6ulevk: make USB configs specific
 to CONFIG_USB

Needed for having USB enabled even without CONFIG_USB_CMD (e.g.
fastboot).

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/mx6ul_14x14_evk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mx6ul_14x14_evk.h b/include/configs/mx6ul_14x14_evk.h
index 10e425d9408..e254b3d03c6 100644
--- a/include/configs/mx6ul_14x14_evk.h
+++ b/include/configs/mx6ul_14x14_evk.h
@@ -222,7 +222,7 @@
 #endif
 
 /* USB Configs */
-#ifdef CONFIG_CMD_USB
+#ifdef CONFIG_USB
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
 #define CONFIG_MXC_USB_PORTSC  (PORT_PTS_UTMI | PORT_PTS_PTW)
 #define CONFIG_MXC_USB_FLAGS   0
-- 
2.25.1

